config/config_tumorSeg.json

```diff
@@ -71,6 +71,7 @@
         "size": [256, 256]
       },
       "ToTensor": {},
+      "StackTensors": {},
```

Comment:
@andreanne-lemay I've added this transformation in [MedicalTorch](https://github.com/neuropoly/medicaltorch/blob/ab/generalization_dataset/medicaltorch/transforms.py#L118) to stack the input tensors so the new loader should not impact your work. Yet I haven't try to train a network, but the [`tests_unet()` ](https://github.com/neuropoly/ivado-medical-imaging/blob/ab/Missing_Modality/testing/test_training_time.py#L35) works. Let me know if you've any troubles!